lib/reek/smells/attribute.rb

```diff
@@ -30,7 +30,7 @@ def self.contexts # :nodoc:
       end
 
       def self.default_config
-        super.merge(Core::SmellConfiguration::ENABLED_KEY => false)
+        super.merge(Core::SmellConfiguration::ENABLED_KEY => true)
```

Comment:
You can enable smells just for a spec run with our handy "with_test_config" helper like here: https://github.com/troessner/reek/blob/develop/spec/reek/smells/nested_iterators_spec.rb#L138